app_dart/test/src/utilities/entity_generators.dart

```diff
@@ -195,12 +195,14 @@ github.CheckRun generateCheckRun(
 
 github.CheckSuite generateCheckSuite(
   int i, {
+  String headBranch = 'main',
```

Comment:
Copied from https://github.com/flutter/cocoon/pull/2430 to get the tests to pass until that merges.